### PR TITLE
update whats new

### PR DESCRIPTION
**Pull request by hugovalente-pm**
##### Summary

Update the What's New section of README

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Review by Ancairon — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -172,20 +172,19 @@ Check the [systemd-journal plugin of Netdata](https://github.com/netdata/netdata
...
-|       Easy Custom<br/>Dashboards        |         Drag and drop charts to create custom dashboards on the fly, while troubleshooting!         |     soon      |                                               in progress                                               |
-|               UCUM Units                |                    Migrate all metrics to the Unified Code for Units of Measure.                    |     soon      |                                               in progress                                               |
-|            Click to Activate            |                          Configure Alerts and Data Collectors from the UI!                          |     soon      |                                               in progress                                               |
+|            Click to Activate            |                          Configure Alerts and Data Collectors from the UI!                          |     soon      | Beta release [v1.45](https://github.com/netdata/netdata/releases/tag/v1.45.0#v1450-dynamic-configuration) - in progress |
```
Comment (ilyam8) [inline README.md:182]: What does "Click to Activate " mean?

Comment (hugovalente-pm) [reply README.md:182]: Activation of configs from the UI, guess it was the term Costa used when he introduced this the first time

Comment (ilyam8) [reply README.md:182]: I'm not sure I understand. The column name is "What", I think that is a feature name. Click what/click where? Active configs - what is that? 

Should it be "Dynamic configuration"? cc @Ancairon

Comment (hugovalente-pm) [reply README.md:182]: I agree, let me change that

**Review by ilyam8 — DISMISSED** [review]

**Comment by ilyam8** [comment]
@hugovalente-pm there are merge conflicts now, can you please resolve them or open a new PR?

**Review by ilyam8 — APPROVED** [review]

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**